transport/http/outbound.go

```diff
@@ -434,10 +434,10 @@ func checkServiceMatch(reqSvcName string, resHeaders http.Header) (bool, string)
 //
 // OpenTracing information must be added manually, before this call, to support context propagation.
 func (o *Outbound) RoundTrip(hreq *http.Request) (*http.Response, error) {
-	return o.roundTrip(hreq, nil /* treq */, time.Now())
+	return o.roundTrip(hreq, nil /* treq */, time.Now(), &transportSender{Client: o.transport.client})
```

Comment:
we are allocating a new `transportSender` for each call -- we should store that in the `*Outbound`

Reply:
ohh very good call. I did not fully understand your previous comment I guess.